### DISCUSSION TITLE
where to increase the chunkSizeLimit?

### DISCUSSION THREAD

**Discussion by GreenInnovator** [GMOD/jbrowse-components#3069] [Q&A] [answered]
When a "too many BAM features" error occurs, can we increase the chunkSizeLimit somewhere in the jbrowse.conf, like in Jbrowse 1?

**Comment by cmdcolin** [comment, marked as answer]
Hi there, it might be interesting to note but we should have completely removed chunkSizeLimit errors in the most recent versions of jbrowse 2

We made chunkSizeLimit infinity in versions 1.6.4, and instead do what we call stats estimation that warns the user if there is a lot of data being loaded but doesn't throw the chunksize limit error https://jbrowse.org/jb2/blog/2022/01/29/v1.6.4-release/

If you are seeing this error on a recent version of jbrowse 2 let me know, but upgrading may fix it. Otherwise, if you are stuck on an older version, you can change chunkSizeLimit through a config on the adapter e.g.


```
   {
      "type": "AlignmentsTrack",
      "trackId": "volvox_alignments",
      "name": "volvox-sorted.bam (ctgA)",
      "assemblyNames": ["volvox"],
      "adapter": {
        "type": "BamAdapter",
        "chunkSizeLimit":1000000000,
        "bamLocation": {
          "uri": "volvox-sorted.bam"
        },
        "index": {
          "location": {
            "uri": "volvox-sorted.bam.bai"
          }
        }
      }
    }
```

**Comment by GreenInnovator** [comment]
Thank you, updating from v1.5 to 1.7 solved it.
